### PR TITLE
enhancement(migrate): delete uesless yaml/sql file

### PR DESCRIPTION
**Pull request by yiminpeng**
#### What type of PR is this?

delete some useless yaml/sql file

#### Which issue(s) this PR fixes:
Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-migrate/src/main/resources/migrate/rbac/V_3_4_0_3__iam_role_permission.yaml [hunk 1]
```diff
@@ -21,4 +21,4 @@ templates:
         data_type: java.lang.Long
       - column_name: creator_id
         value: ${CREATOR_ID}
-        data_type: java.lang.Long
\ No newline at end of file
+        data_type: java.lang.Long
```
Comment (MarkPotato777) [inline server/odc-migrate/src/main/resources/migrate/rbac/V_3_4_0_3__iam_role_permission.yaml:24]: It is better not to involve empty line change if not necessary.

File: server/odc-migrate/src/main/resources/migrate/common/V_3_4_0_12__add_data_masking.sql
Comment (MarkPotato777) [inline server/odc-migrate/src/main/resources/migrate/common/V_3_4_0_12__add_data_masking.sql:1]: `V_3_4_0_2__iam_permission.yaml` and `V_3_4_0_3__iam_role_permission.yaml` are both deprecated too.

**Review by MarkPotato777 — COMMENTED** [review]
Please not the PR title. This PR seems not like a feature but an enhancement

**Review by MarkPotato777 — COMMENTED** [review]
Some unit tests failed, please check.

File: server/odc-migrate/src/main/resources/migrate/common/V_3_4_0_12__add_data_masking.sql
Comment (yiminpeng) [reply server/odc-migrate/src/main/resources/migrate/common/V_3_4_0_12__add_data_masking.sql:1]: OK

Comment (yiminpeng) [reply server/odc-migrate/src/main/resources/migrate/common/V_3_4_0_12__add_data_masking.sql:1]: OK

**Review by MarkPotato777 — APPROVED** [review]
LGTM

**Pull request closed without merging**